.github/workflows/docs.yml

```diff
@@ -63,5 +63,4 @@ jobs:
         run: |
             cd docs
             poetry run make html
-            poetry run make doctest
-            poetry run pytest  --doctest-glob=*.rst
\ No newline at end of file
+            poetry run make doctest
```

Comment:
This in addition to changes in 587fa07429aca2d06d933e9af9c5d636ab99aacc are due to some error with cvxpy.

https://github.com/vprusso/toqito/actions/runs/10516305795/job/29138240242?pr=754#step:5:3507